packages/dd-trace/src/appsec/index.js

```diff
@@ -53,7 +52,7 @@ function enable (_config) {
 
     RuleManager.loadRules(_config.appsec)
 
-    remoteConfig.enableWafUpdate(_config.appsec)
+    global._ddtrace?._modules?.rc?.module?.enableWafUpdate(_config.appsec)
```

Comment:
Wouldn't diagnostics channel be more appropriate at that point?

Reply:
Otherwise the common pattern in other places would be a noop, although I'm starting to lean towards DC more and more as a way to decouple things in a way that is more well, decoupled, and also makes for easier bundling and detection of private access.